docs/General/install.rst

```diff
@@ -278,13 +280,17 @@ When the Ninja command is finished running, you will find an installer in :file:
 Windows
 -------
 
-Msys2 is an available environment for compiling Freeciv21. Microsoft Windows Visual C is under development.
+MSYS2 is an available environment for compiling Freeciv21. Microsoft Windows is supported with :file:`clang`
+in :doc:`Visual Studio <../Contributing/visual-studio>`.
 
-Freeciv21 currently supports building and installing using the Msys2 environment. Build instructions for
-Msys2 versions are documented in :doc:`../Contributing/msys2`. Alternately you can visit
+The Freeciv21 community primarily supports building and installing on Windows using the MSYS2 environment.
+While Microsoft Visual Studio can be used, we do not package binaries via Visual Studio due to licensing
+`constraints <https://www.gnu.org/licenses/gpl-faq.en.html#WindowsRuntimeAndGPL>`_.

```

Comment:
These two paragraph appear to contradict each other: in the first one MSYS2 is "available" and VS is "supported" while in the second MSYS2 is "supported" and VS "can be used".